content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Authorization on Ingress Gateway
+description: Shows how to set up access control on ingress gateway.
+weight: 50
+keywords: [security,access-control,rbac,authorization,ingress,ip,whitelist,blacklist]
+---
+
+This task shows you how to enforce access control on an Istio ingress gateway
+using an authorization policy. 
+
+The authorization policy supports IP-based whitelists or blacklists and
```

Comment:
nit: since this is being revised, perhaps, adopt allowlist/denylist instead for a more inclusive language?

Reply:
Thanks, updated to use allow list and deny list.